brianlong/www.validators.app
# 178676857 deadlocks for hetzner script

#### What's this PR do?
second attempt to remove the deadlock error

#### How should this be manually tested?
just run the gather and score daemons and see if they work properly and doesn't take too long

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/178676857

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
